src/fft_fftw.cpp

```diff
@@ -92,35 +92,35 @@ void Fftw::init(unsigned width, unsigned height, unsigned num_of_feats, unsigned
 #endif
 #ifdef BIG_BATCH
     // FFT inverse all scales
-    {
-        ComplexMat in_i_all(m_height, m_width, m_num_of_feats * m_num_of_scales);
-        cv::Mat out_i_all = cv::Mat::zeros(m_height, m_width, CV_32FC(m_num_of_feats * m_num_of_scales));
+    if (m_num_of_scales > 1) {
```

Comment:
Why this if? For big batch, m_num_of_scales is always > 1.

Reply:
This is safety check, for the possibility that the someone would set the p_num_scales to 1 in kcf.h and wanted to do big batch mode. In that case it is unnecessary to create big batch plans.

Reply:
So it is not safety check but unnecessary optimization. If the check is not there, the program will still work correctly, right?

Reply:
Yes. I worded that incorrectly.